CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 
 #### Bug Fixes
 * pickle filter: support relative paths ([#962](https://github.com/cucumber/cucumber-js/pull/962) Marco Muller)
+* cucumber-expressions: Upgrade from 5.0.3 to [5.0.6](https://github.com/cucumber/cucumber/blob/master/cucumber-expressions/CHANGELOG.md#506---2017-11-28)
+* tag-expressions: Upgrade from 1.0.1 to [1.1.0](https://github.com/cucumber/cucumber/blob/master/tag-expressions/CHANGELOG.md#110---2017-11-28)
```

Comment:
Please create a new section in the changelog (unreleased) for this changes

Reply:
Ah crap I thought that’s what I did. I always make an empty section for the next release when I make a release. That way contributors have a place to dump comments. They don’t have to figure out what the section formatting is like.

Can you do this for future releases? Example: https://github.com/cucumber/cucumber/blob/master/gherkin/CHANGELOG.md

Reply:
Yeah I like that idea but just haven't been disciplined enough about following it